lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -228,25 +234,57 @@ def sha256sum(filename):
     return hex_digest
 
 
+# There is a PR open to merge fail_json_aws this into the standard module code;
+# see https://github.com/ansible/ansible/pull/23882
+def fail_json_aws(module, exception, msg=None):
```

Comment:
Now that the AnsibleAWSModule is included in this PR, I think this function can go away.